### PR TITLE
feat(start_nominating): add section how to change nominees

### LINKED ISSUE #23: [start_nominating] add change nominations explanation
https://docs.hydradx.io/start_nominating/#03-check-the-status-of-your-nominations

Please add information how to change nominations by adding/removing validators.
https://support.polkadot.network/support/solutions/articles/65000170911-how-do-i-change-my-validators-

@vgantchev

### PR DESCRIPTION
**Pull request by SimonKraus**
fixed #23

### CODE REVIEW

File: docs/start_nominating.md [hunk 1]
```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
 
 <div style={{textAlign: 'center'}}>
   <img src={useBaseUrl('/nominator-guide/nominate-validator-4.png')} />
-</div>
+</div>
```
Comment (vgantchev) [inline docs/start_nominating.md:126]:
```suggestion
</div>
```

File: docs/start_nominating.md [hunk 2]
```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
...
-</div>
+</div>  
 
 :::note
 
 Make sure that you revisit your nominations once in a while. It is possible that some of your validators change their commission percentage which will have a negative impact on your rewards. By checking the status of your nominations frequently you will be able to react by updating the list of your nominated validators.
 
 :::
 
+## 04 Change Nominees
```
Comment (vgantchev) [inline docs/start_nominating.md:134]: Adjust your nominations

(let's not use Camel Case to remain consistent with other titles)

File: docs/start_nominating.md [hunk 3]
```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
...
+
+There may be several circumstances where you'd like to change your nominations or remove specific ones.
```
Comment (vgantchev) [inline docs/start_nominating.md:136]: If some of your validators become oversubscribed or change their commission, you might want to adjust your nominations.

To do so, open Polkadot/apps and navigate to:

File: docs/start_nominating.md [hunk 4]
```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
...
 
+## 04 Change Nominees
+
+There may be several circumstances where you'd like to change your nominations or remove specific ones.  
+This could be due to oversubscription of a validator or if he changes the commission rate.  
+
+To do so open Polkadot/apps and navigate over to:  
+*Network* > *Staking* > *Account actions*
+
+Hit the three dots at the end of your account details and click _Set nominees_.
```
Comment (vgantchev) [inline docs/start_nominating.md:142]:
```suggestion
Click on the three dots next to your account details ans select _Set nominees_.
```

File: docs/start_nominating.md [hunk 5]
```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
...
+*Network* > *Staking* > *Account actions*
+
+Hit the three dots at the end of your account details and click _Set nominees_.  
+
+<div style={{textAlign: 'center'}}>
+  <img src={useBaseUrl('/nominator-guide/nominate-set-nominees.png')} />
+</div>
+
+In the following window, that may already feel familiar, you can remove validators and/or add new ones.  
+This step can be done on the fly, there is no need to stop nominating. It will be respected for the next era.
```
Comment (vgantchev) [inline docs/start_nominating.md:149]:
```suggestion
Adjusting your nominations can be done on the fly, there is no need to stop nominating. The changes will be applied when the following era begins (24h).
```

File: docs/start_nominating.md [hunk 6]
```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
...
+To do so open Polkadot/apps and navigate over to:  
+*Network* > *Staking* > *Account actions*
+
+Hit the three dots at the end of your account details and click _Set nominees_.  
+
+<div style={{textAlign: 'center'}}>
+  <img src={useBaseUrl('/nominator-guide/nominate-set-nominees.png')} />
+</div>
+
+In the following window, that may already feel familiar, you can remove validators and/or add new ones.
```
Comment (vgantchev) [inline docs/start_nominating.md:148]:
```suggestion
You will see the nominating UI where you can adjust your nominations by removing and adding validators.
```

**Review by vgantchev — COMMENTED** [review]
Thanks for picking this up @SimonKraus 
It is already good as-is but I am leaving some suggestions which mostly have to do with style and consistency

btw I notice a lot of nbsp at the end of lines, could you please set your editor settings to strip these?

File: docs/start_nominating.md (hunk 1 above)
Comment (SimonKraus) [reply docs/start_nominating.md:126]: Double-space is a linebreak in markdown (<br>)

**Comment by SimonKraus** [comment]
@vgantchev  feedback is included in the last commit

**Comment by vgantchev** [comment]
LGTM 👍

**Review by vgantchev — APPROVED** [review]

**Review by jak-pan — APPROVED** [review]

**Pull request merged**